schema/hescore_json.schema.js

```diff
@@ -0,0 +1,26 @@
+const JSONSchema = require('./_base.schema.js');
+
+// TODO: There is a PR open to make this change to the base schema: https://github.com/NREL/hescore-hpxml/pull/228
+//  When that has been merged and hescore_json_schema.js has been updated, we can remove this code.
+JSONSchema.properties.address.properties.state.enum = [
+    "AA", "AE", "AL", "AK", "AP", "AS", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "FM", "GA", "GU", "HI",
+    "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MH", "MI", "MN", "MP", "MS", "MO", "MT", "NE",
+    "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "PR", "PW", "RI", "SC", "SD", "TN", "TX",
+    "UT", "VT", "VA", "WA", "WV", "WI", "WY"
+]
+
+/**
+ * Assessment type is under PNNL's control, so we don't want NREL to be responsible for validating it.
```

Comment:
@charliepnnl - Do we maybe want to ask NREL to make this a generic "string"?  Right now, assessment types are tracked more directly using our database, which is necessary for tracking "official" types, access, etc

If so, I propose we add a similar TODO as above, which would in the near future render this file obsolete too